### DISCUSSION TITLE
Add padding above url bar

### DISCUSSION THREAD

**Discussion by zenwav** [Floorp-Projects/Floorp#941] [Feature Request]
At present, the lack of padding above the URL bar when the title bar is disabled makes it impossible to grab the browser window and move it around. A potential solution could be to add some padding above the URL bar, similar to the design in Midori.
![23February2024-Time=01;;29;;30](https://github.com/Floorp-Projects/Floorp/assets/24970562/0b18dbdb-e214-4662-952e-b461fbe40e4f)

**Comment by regularentropy** [comment]
You use tabbar at bottom?

> At present, the lack of padding above the URL bar when the title bar is disabled makes it impossible to grab the browser windows move it around. A potential solution could be to add some padding above the URL bar, similar to the design in Midori. ![23February2024-Time=01;;29;;30](https://private-user-images.githubusercontent.com/24970562/307133480-0b18dbdb-e214-4662-952e-b461fbe40e4f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg2MzU5MDksIm5iZiI6MTcwODYzNTYwOSwicGF0aCI6Ii8yNDk3MDU2Mi8zMDcxMzM0ODAtMGIxOGRiZGItZTIxNC00NjYyLTk1MmUtYjQ2MWZiZTQwZTRmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIyVDIxMDAwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIyMzFiZWI2Y2ZjMDgzYmJiN2U5OWVmYmJkOTEyZjkwMTk5NDI4NzNjYzdkMjJkNDEwYmJiZjE4MTk1MTViMTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.kK6V9A5V11Psgw98bTgM9Xadk1othN9k3s3lAZ_01Kg)

**Reply by zenwav** [reply]
No, I use vertical tabs

**Reply by regularentropy** [reply]
Ok, thanks. I'll work on it.

**Reply by regularentropy** [reply]
@averagePedestrian https://github.com/Floorp-Projects/Floorp-core/pull/58

**Comment by lnkd61** [comment]
Could we have this as an option rather than forced? I already have dragspace in the layout I created and so the extra padding at the top is redundant.

**Reply by regularentropy** [reply]
https://github.com/Floorp-Projects/Floorp-core/pull/59

**Comment by JimFlasch** [comment]
Yeah - should have been optional from the minute it was added so those of us who don't care for it don't have to have it!

**Comment by JimFlasch** [comment]
Seems that the padding IS optional - but hiding it (with Default or Hide tabs on Horizontal Tab Bar under Tabs on the Preferences/Design page) makes the Minimize, Restore Up/Down and Close buttons disappear! Can that be changed?
